hil/model.py

```diff
@@ -258,6 +258,14 @@ def ensure_legal_operation(self, nic, op_type, channel):
 
         return
 
+    def get_capabilities(self):
+        """Returns the list of capabilities supported by a switch
+
+        default behaviour is to return a list with capabilities supported by
+        most switches"""
+
+        return ['nativeless-trunk-mode']
```

Comment:
I think I'd prefer to not have a default implementation, and mandate that drivers be explcit. Or, if we do want to have a default, it should return the empty list, so a driver author has to explicitly say their driver can do something.